internal/fs/fs_test.go

```diff
@@ -21,25 +22,35 @@ func TestHasFilepathPrefix(t *testing.T) {
 	}
 	defer os.RemoveAll(dir)
 
+	// dir2 is the same as dir but with different capitalization on Windows to
+	// test case insensitivity
```

Comment:
As noted in the issue discussion, with the exception of the volume name (i guess?), case insensitivity isn't a _windows_ property, but actually a _filesystem_ property. (And within that, I guess, a property of how [`CreateFile`()](https://msdn.microsoft.com/en-us/library/aa363858(v=vs.85).aspx) is called - whether `FILE_FLAG_POSIX_SEMANTICS` is set). This all suggests to me that some Windows configurations _could_ fail on this test.

That seems like a rare case, though, and this does add coverage for case insensitivity in a way that makes our tests more robust. So, I'll accept it. However, please expand the comment a bit to note that failure _may_ be possible on Windows.